lib/ransack/adapters/mongoid/base.rb

```diff
@@ -8,6 +8,10 @@ module Base
         extend ActiveSupport::Concern
 
         included do
+          class_attribute :_ransackers
+          class_attribute :_ransack_aliases
+          self._ransackers ||= {}
+          self._ransack_aliases ||= {}
```

Comment:
Using `class_attribute` here makes much of Mongoid's implementation identical to that of Active Record's. :+1: